packages/scripts/src/admin-proposals/proposeAdminTransactions.js

```diff
@@ -0,0 +1,68 @@
+// Description:
+// - Propose admin transactions manually.
+
+require("dotenv").config();
+const { getWeb3ByChainId } = require("@uma/common");
+const { setupGasEstimator, proposeAdminTransactions } = require("./utils");
+const { REQUIRED_SIGNER_ADDRESSES } = require("../utils/constants");
+
+// Hardcode this array with admin transaction objects. Tip: you can console.log the `adminProposalTransactions` objects
+// in the other admin-proposals scripts.
+const adminProposals = [
+  // Sending a Finder.changeImplementationAddress transaction to the Polygon finder via the OracleRootTunnel
+  {
+    to: "0x4F490F4835B3693A8874aee87D7CC242c25DCCAf",
+    value: 0,
+    data:
+      "0x6296932d00000000000000000000000009aea4b2242abc8bb4bb78d537a67a245a7bec640000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000004431f9665e4f7261636c650000000000000000000000000000000000000000000000000000000000000000000000000000bed4c1fc0fd95a2020ec351379b22d8582b904e300000000000000000000000000000000000000000000000000000000",
+  },
+  // The next three transactions will register a new contract on Ethereum. In order:
+  // 1. Add the Governor as a contract creator that can register new contracts
+  {
+    to: "0x3e532e6222afe9Bcf02DCB87216802c75D5113aE",
+    value: 0,
+    data:
+      "0x74d0a6760000000000000000000000000000000000000000000000000000000000000001000000000000000000000000592349f7dedb2b75f9d4f194d4b7c16d82e507dc",
```

Comment:
Would it be better to do the encoding in line to make it clear what's happening?

Reply:
Eh that would add a ton of extra imports to this script I want to keep it lightweight for now. I don't anticipate using this script much but if we do we can refactor and add more features